src/main/java/io/confluent/idesidecar/restapi/kafkarest/controllers/RelationshipUtil.java

```diff
@@ -0,0 +1,118 @@
+            SIDECAR_HOST, clusterId, topicId
+        )).build();
+  }
+
+  public static Relationship forAllPartitionReassignments(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/topics/-/partitions/-/reassignment".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forTopicConfigs(String clusterId, String topicId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/topics/%s/configs".formatted(
+            SIDECAR_HOST, clusterId, topicId
+        )).build();
+  }
+
+  public static Relationship forController(String clusterId, int brokerId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/brokers/%d".formatted(
+            SIDECAR_HOST, clusterId, brokerId
+        )).build();
+  }
+
+  public static Relationship forAcls(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/acls".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forBrokers(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/brokers".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forBrokerConfigs(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/broker-configs".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forConsumerGroups(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/consumer-groups".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forTopics(String clusterId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/topics".formatted(
+            SIDECAR_HOST, clusterId
+        )).build();
+  }
+
+  public static Relationship forTopic(String clusterId, String topicId) {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters/%s/topics/%s".formatted(
+            SIDECAR_HOST, clusterId, topicId
+        )).build();
+  }
+
+  public static Relationship forClusters() {
+    return Relationship.builder().related(
+        "%s/internal/kafka/v3/clusters".formatted(
+            SIDECAR_HOST
+        )).build();
+  }
+
+  public static Crn getClusterCrn(String clusterId) {
+    return new Crn(
+        CRN_AUTHORITY,
+        Crn.newElements("kafka", clusterId),
+        false
+    );
+  }
+
+  public static Crn getTopicCrn(String clusterId, String topicName) {
+    return new Crn(
+        CRN_AUTHORITY,
+        Crn.newElements("kafka", clusterId, "topic", topicName),
+        false
+    );
+  }
```

Comment:
Should these be in `Crn` like the other methods? Yes, these are not CCloud related, but that's okay.

Reply:
Removed these methods. We decided to tackle constructing CRNs in a follow up.